scripts/git/pre-commit

```diff
@@ -22,7 +22,7 @@ changed_cr_files=$(git diff --cached --name-only --diff-filter=ACM | grep '\.cr$
 
 if [ -x bin/crystal ]; then
   # use bin/crystal wrapper when available to run local compiler build
-  exec bin/crystal tool format --check $changed_cr_files >&2
+  exec bin/crystal tool format --check "$changed_cr_files" >&2
```

Comment:
This is wrong, splatting is being prevented

Reply:
Actually, the current variant is also wrong. Paths with white space or unescaped glob characters would break. Not that there are any, at the moment.
I'll revert this and maybe we can fix it in a follow-up.